lib/tasks/assets.js

```diff
@@ -98,5 +89,16 @@ module.exports = Task.extend({
       stopOnErr: true,
       limit: 2
     });
+  },
+
+  deleteExcluded: function(dir, config){
```

Comment:
Thanks, @achambers. Updated the reference. It's pretty bad that our tests are in sufficient to catch such a thing. We really need the integration test that @danshultz started in #117.